### PR TITLE
adds docs for speed curve calibration

### PR DESCRIPTION
**Pull request by devinaconley**
This closes https://github.com/monoDriveIO/monodrive-client/issues/107 and corresponds with this C++ client PR https://github.com/monoDriveIO/monodrive-client/pull/113

### CODE REVIEW

File: docs/monoDrive_home/Vehicle-PhysX.md [hunk 1]
```diff
@@ -48,4 +48,21 @@ The Steering Curve enables the user to limit the vehicle’s maximum turn angle
...
+Once the vehicle physics model has been configured, it can be compared and validated against
+real world data.
+
+For example, see the [speed curve C++ example](https://github.com/monoDriveIO/monodrive-client/tree/master/examples/cpp/lane_follower#lane-follower---speed-curve) which uses a PID controller to reproduce a target
+time series of speed values.
+
+Here is a comparison between the original (mock) data and the curve reproduced in simulation.
+
+<p class="img_container">
+<img class="wide_img" src="../img/speed_curve_reproduced.png" />
```
Comment (bradleynull) [inline docs/monoDrive_home/Vehicle-PhysX.md:65]: This isn't linking to the correct image location. Should move the image to `monoDrive_home/img` or change this to link to the base `docs/imgs` directory.

**Review by bradleynull — CHANGES_REQUESTED** [review]
@devinaconley Just need to fix the image.

File: docs/monoDrive_home/Vehicle-PhysX.md (hunk 1 above)
Comment (devinaconley) [reply docs/monoDrive_home/Vehicle-PhysX.md:65]: Fixed good catch!

**Review by bradleynull — APPROVED** [review]
@devinaconley Looks good to go! Approved!

**Pull request merged**